### PR TITLE
feat(translate): add translate text sample and test

### PR DESCRIPTION
**Pull request by OremGLG**
Add Translate Text sample

Fixes Internal: b/421269888

Added empty fixture class for future implementation.

### CODE REVIEW

File: translate/api/Translate.Samples/Text.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/*
```
Comment (cy-yun) [inline translate/api/Translate.Samples/Text.cs:1]: file name should be TranslateTextSample.cs

File: translate/api/Translate.Samples/Text.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    /// </summary>
+    /// <param name="targetLanguage">The ISO 639 language code to translate the text into.</param>
+    /// <param name="sourceLanguage">
+    /// Optional. The ISO 639 language code of the input text. If null, the API will attempt
+    /// to detect the source language automatically.
+    /// </param>
+    /// <param name="text">The text to translate.</param>
+    /// <returns>
+    /// An Execute object representing the completed workflow execution.
+    /// </returns>
```
Comment (cy-yun) [inline translate/api/Translate.Samples/Text.cs:34]: is there a more readable format you can follow for this documentation?

File: translate/api/Translate.Samples/Text.cs [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    public async Task<TranslationResult> TranslateText(
+        string targetLanguage,
+        string? sourceLanguage,
+        string text)
+    {
+        // Create TranslationClient.
+        TranslationClient client = await TranslationClient.CreateAsync();
+
+        // Translate text.
```
Comment (cy-yun) [inline translate/api/Translate.Samples/Text.cs:43]: redundant

File: translate/api/Translate.Samples/Text.cs [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        TranslationResult translationResult = client.TranslateText(text, targetLanguage, sourceLanguage);
+
+        // Print results.
```
Comment (cy-yun) [inline translate/api/Translate.Samples/Text.cs:46]: redundant

File: translate/api/Translate.Samples.Test/TextTests.cs [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        _sample = new TranslateTextSample();
+    }
+
+    [Fact]
+    public async Task TranslateTest()
+    {
+        TranslationResult translationResult = await _sample.TranslateText("en", "es", "¡Hola amigos y amigas!");
+
+        Assert.Contains("Hello friends!", translationResult.TranslatedText);
+    }
```
Comment (cy-yun) [inline translate/api/Translate.Samples.Test/TextTests.cs:37]: add a test where source language is null
